src/rustllvm/RustWrapper.cpp

```diff
@@ -675,10 +675,12 @@ extern "C" LLVMMetadataRef LLVMRustDIBuilderCreateCompileUnit(
                                          fromRust(Kind)));
 }
 
-extern "C" LLVMMetadataRef
-LLVMRustDIBuilderCreateFile(LLVMRustDIBuilderRef Builder, const char *Filename,
-                            const char *Directory) {
-  return wrap(Builder->createFile(Filename, Directory));
+extern "C" LLVMMetadataRef LLVMRustDIBuilderCreateFile(
+    LLVMRustDIBuilderRef Builder,
+    const char *Filename, size_t FilenameLen,
+    const char *Directory, size_t DirectoryLen) {
+  return wrap(Builder->createFile({Filename, FilenameLen},
```

Comment:
Would it be perhaps be better to be explicit about what type (`StringRef`) is being created here? I'd find `StringRef { Filename, FilenameLen }` significantly easier to read (applies across this file).

Reply:
Repalced with `StringRef(Name, NameLen)` which I feel is more common in LLVM itself.